From 5fb1ae5049de5318a064a63f3d974e118ae3a448 Mon Sep 17 00:00:00 2001
From: Michal Simek <michal.simek@xilinx.com>
Date: Tue, 14 Jan 2020 11:20:27 -0800
Subject: [PATCH 0248/1852] drm: xilinx: Kconfig: Add missing DMADEVICES
 dependencies

commit 66f5e54b2c3b0cbcf3c23947aa2ab2b15e00802a from
https://github.com/Xilinx/linux-xlnx.git

Several drivers selecting Xilinx DMA drivers but not all dependencies
are properly covered.

WARNING: unmet direct dependencies detected for XILINX_DPDMA
  Depends on [n]: DMADEVICES [=n] && XILINX_DMA_ENGINES [=n]
  Selected by [y]:
  - DRM_ZYNQMP_DPSUB [=y] && HAS_IOMEM [=y] && ARCH_ZYNQMP [=y] && OF [=y] && DRM_XLNX [=y] && COMMON_CLK [=y]

Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/gpu/drm/xlnx/Kconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/gpu/drm/xlnx/Kconfig b/drivers/gpu/drm/xlnx/Kconfig
index d6101a421aa7..f2ca801850c5 100644
--- a/drivers/gpu/drm/xlnx/Kconfig
+++ b/drivers/gpu/drm/xlnx/Kconfig
@@ -33,6 +33,8 @@ config DRM_XLNX_BRIDGE_DEBUG_FS
 config DRM_ZYNQMP_DPSUB
 	tristate "ZynqMP DP Subsystem Driver"
 	depends on ARCH_ZYNQMP && OF && DRM_XLNX && COMMON_CLK
+	select DMADEVICES
+	select XILINX_DMA_ENGINES
 	select XILINX_DPDMA
 	select GENERIC_PHY
 	help
-- 
2.31.1

